tensorflow/examples/udacity/.gitignore

```diff
@@ -0,0 +1,5 @@
+notMNIST.pickle
```

Comment:
Sorry I have not made my way through the other classes.
So is it the case that it's better to regex this as *.pickle?